root/search/results-work.tt

```diff
@@ -10,7 +10,7 @@
                             <th>[%- l('Artists') -%]</th>
                             <th>[%- l('ISWC') -%]</th>
                             <th>[%- l('Type') -%]</th>
-                            <th>[%- l('Language') -%]</th>
+                            <th>[%- l('Languages') -%]</th>
```

Comment:
I think we use "Lyrics Languages" as the label everywhere else.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/593/commits/4e37e35ffeb3f3d792f4d5840a47c8172406280a (to be squashed on merge)